arch/arm/src/stm32/stm32_wwdg.c

```diff
@@ -591,8 +591,8 @@ static int stm32_settimeout(FAR struct watchdog_lowerhalf_s *lower,
   priv->fwwdg  = fwwdg;
   priv->reload = reload;
 
-  wdinfo("wdgtb=%d fwwdg=%d reload=%d timeout=%d\n",
-         wdgtb, fwwdg, reload, priv->timeout);
+  wdinfo("wdgtb=%d fwwdg=%u reload=%u timeout=%u\n",
+         wdgtb, (unsigned)fwwdg, (unsigned)reload, (unsigned)priv->timeout);
```

Comment:
i prefer to use PRIu32 etc where it's easy.
i don't feel strong on this though.